src/Data/pisaClient.ts

```diff
@@ -1,15 +1,33 @@
-import { Obj, createRestApiClient, currentLanguage, load } from 'scrivito'
+import {
+  Obj,
+  createRestApiClient,
+  currentLanguage,
+  currentUser,
+  load,
+} from 'scrivito'
 import { isHomepage } from '../Objs/Homepage/HomepageObjClass'
 
-export async function pisaUrl(): Promise<string | null> {
+async function pisaBaseUrl(): Promise<string | null> {
   if (import.meta.env.FORCE_LOCAL_STORAGE) return null
+  if (!(await load(() => currentUser()))) return null
```

Comment:
* [ ] FYI `isUserLoggedIn` is the simpler alternative here, see https://docs.scrivito.com/isuserloggedin-a84bd9ef936b26f6#isUserLoggedIn-vs-currentUser

Reply:
But `isUserLoggedIn` only works, if `Scrivito.configure` was called previously. This is not always the case with `pisaBaseUrl()`.

Reply:
If this is true either the docs are wrong or the SDK has a bug:

> the application logic never has to wait for the result of isUserLoggedIn, and is not required to handle the case that the value of isUserLoggedIn may change.